pkg/qdr/qdr.go

```diff
@@ -750,6 +750,7 @@ type BridgeConfigDifference struct {
 	TcpConnectors  TcpEndpointDifference
 	HttpListeners  HttpEndpointDifference
 	HttpConnectors HttpEndpointDifference
+	SslProfiles    []string
```

Comment:
Do we need to track deleted ssl profiles as well? I.e. what happens if a tls enabled service is unexposed then re-exposed?